### DISCUSSION TITLE
Slow Navigation in Next js 14

### DISCUSSION THREAD

**Discussion by code-harsh-ui** [community/community#121355] [Programming Help]
### Body

Hi everyone, I'm having a problem with navigation links in Next.js. Whenever I try to switch between pages, it takes too long to navigate. For example, if I switch from the home page to the contact or about page, and then back to the home page, it takes too long. I'm very frustrated right now. Please, can anyone help me with this? You can check it yourself by visiting the link - https://videhk.netlify.app/

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hey @code-harsh-ui!

Did you end up fixing the issue? It seemed fine to me. The only thing is the blog page said the request could not be completed (in the console there's an error message about it as well)
